visual-js/visual-snapshots/jest.config.cjs

```diff
@@ -0,0 +1,20 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
+module.exports = {
+  // [...]
+  preset: 'ts-jest/presets/default-esm', // or other ESM presets
+  moduleNameMapper: {
+    '^(\\.{1,2}/.*)\\.js$': '$1',
+  },
+  testPathIgnorePatterns: ['/build/'],
+  transform: {
+    // '^.+\\.[tj]sx?$' to process js/ts with `ts-jest`
+    // '^.+\\.m?[tj]sx?$' to process js/ts/mjs/mts with `ts-jest`
+    '^.+\\.tsx?$': [
+      'ts-jest',
+      {
+        useESM: true,
+      },
+    ],
+  },
+  setupFiles: ['<rootDir>/setEnvVars.ts'],
```

Comment:
The file has a bit cryptic file name when looking at it outside of Jest's context. Can we rename it to something like `jest.setup.ts`, or any name that would signify that it is used solely for tests?

Reply:
The file has been deleted